experiments/defaults.py

```diff
@@ -154,13 +157,15 @@ def default_tokenize(
             cache_path=this_output_path(),
             tokenizer=ensure_versioned(tokenizer),
             format=format,
+            sample_count=ensure_versioned(sample_count) if sample_count is not None else None,
```

Comment:
e.g. let's say we didn't have default_tokenize, we might be encouraged to improve our tokenization interface to be more fluent and maybe write something like:

Tokenizer::from_hf(...).take(x).to_step()

This just feels more directly usable than bouncing through default_tokenize

Reply:
fluent builder seems nice